npm/builder.js

```diff
@@ -487,6 +490,17 @@ Builder.prototype.compile = function() {
   fs.createReadStream(asciidoctorCSSFile).pipe(fs.createWriteStream('build/asciidoctor.css'));
 };
 
+Builder.prototype.replaceUnsupportedFeatures = function() {
+  log.title('Replace unsupported features');
+  var path = 'build/asciidoctor-core.js';
+  var data = fs.readFileSync(path, 'utf8');
+  log.debug('Replace sub! with sub');
+  data = data.replace(/\['\$sub!'\]/gi, '[\'$sub\']');
```

Comment:
The `i` flag is not necessary here since the method name is always lowercase. I also recommend combining these two replacements into a single call to avoid scanning the input twice.

```js
data = data.replace(/(\['\$g?sub)!(?='\])/g, "$1")
```

or a bit less fancy:

```js
data = data.replace(/\['(\$g?sub)!'\]/g, "['$1']")
```